packages/html/lib/index.js

```diff
@@ -1,11 +1,15 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-exports.minify = void 0;
+exports.minifySync = exports.minify = void 0;
```

Comment:
Also this one.
Can you add `tsc -d` to the package.json, too?

Reply:
Sorry, what does that mean? This line is already tsc -d auto-converted.

Reply:
Nevermind, we already have it